awx/__init__.py

```diff
@@ -34,10 +35,8 @@ def version_file():
 
 
 try:
-    import pkg_resources
-
-    __version__ = pkg_resources.get_distribution('awx').version
-except pkg_resources.DistributionNotFound:
```

Comment:
LOL, this line errors out with a `NameError` because it never exists prior to the import.